terraform.tf

```
@@ -0,0 +1,2 @@
+terraform {
+}
```

```suggestion
terraform {
  backend "s3" {}

  required_version = ">= 1.3.5"

  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = ">= 3.72.0"
    }
  }
}

provider "aws" {
  region = var.region
}
```
Ideally here you should specify the providers and also each provider custom config if need (i.e aws region)